azure/group.sh

```diff
@@ -7,8 +7,21 @@ fn azure_group_create(name, location) {
 	azure group create --name $name --location $location
 }
 
+# azure_group_get_id get a `resource group` id.
+# `name` is the resource group name
+#
+# This function return resource group id
+fn azure_group_get_id(name) {
+        resp <= (
+                azure group show --name $name --json
```

Comment:
The `azure` cli doesn't exit with error in case of resource group not found? If yes, then this will abort the script. That's your intent?